src/Compilers/Core/CodeAnalysisTest/PEWriter/InstructionOperandTypesTests.cs

```diff
@@ -43,8 +43,8 @@ public void OperandTypes()
                 }
             }
 
-            AssertEx.Equal(OneByteOperandTypes, InstructionOperandTypes.OneByte);
-            AssertEx.Equal(TwoByteOperandTypes, InstructionOperandTypes.TwoByte);
+            AssertEx.Equal(OneByteOperandTypes, InstructionOperandTypes.OneByte.ToArray());
+            AssertEx.Equal(TwoByteOperandTypes, InstructionOperandTypes.TwoByte.ToArray());
```

Comment:
Would prefer we add an overload to `AssertEx` that takes `ROSpan<T>` types vs. calling `ToArray` here.